Tasks/GradleV2/task.json

```diff
@@ -322,6 +322,45 @@
             "defaultValue": "false",
             "groupName": "CodeAnalysis",
             "helpMarkDown": "Use the PMD Java static analysis tool to look for bugs in the code. Results are uploaded as build artifacts."
+        },
+        {
+            "name": "spotBugsAnalysisEnabled",
+            "aliases": [
+                "spotBugsAnalysis"
+            ],
+            "type": "boolean",
+            "label": "Run SpotBugs",
+            "required": true,
+            "defaultValue": "false",
+            "groupName": "CodeAnalysis",
+            "helpMarkDown": "Enable this option to run spotBugs. This plugin works with Gradle v5.6 or later. [More info](https://spotbugs.readthedocs.io/en/stable/gradle.html#use-spotbugs-gradle-plugin)"
+        },
+        {
+            "name": "spotBugsGradlePluginVersionChoice",
```

Comment:
I don't see where this input is used in your code. What it was added for?
The same question for the _spotbugsGradlePluginVersion_ input.

Reply:
@EzzhevNikita 
You can see it in this spotbugsTool file in this PR https://github.com/microsoft/azure-pipelines-tasks/pull/14618/files
Those inputs are used in spotbugsTool. 
I implemented the spotbugs plugin in the same way as the sonarqube plugin.
![image](https://user-images.githubusercontent.com/79705997/111767471-d5113700-88b7-11eb-9aeb-0c46a47de921.png)
![image](https://user-images.githubusercontent.com/79705997/111767577-f4a85f80-88b7-11eb-8518-bd5417b3b727.png)

Reply:
Got it, thank you for the explanation